mindmeld/components/entity_resolver.py

```diff
@@ -335,22 +429,8 @@ def predict(self, entity):
         Returns:
             (list): The top 20 resolved values for the provided entity.
         """
-        if isinstance(entity, (list, tuple)):
-            top_entity = entity[0]
-            entity = tuple(entity)
-        else:
-            top_entity = entity
-            entity = tuple([entity])
 
-        if self._is_system_entity:
-            # system entities are already resolved
-            return [top_entity.value]
-
-        if self._no_canonical_entity_map:
-            return []
-
-        if not self._use_text_rel:
-            return self._predict_exact_match(top_entity)
+        top_entity = entity[0]
```

Comment:
Do we need to handle the case were `entity ` is just a single entity? If there is impossible, lets change the name of the arg from `entity ` to something similar to `nbest_entities` (plural)

Reply:
I think entity can be single or multiple entities, however, they are converted into a tuple before passing to `_predict()`. I think it makes sense to change the term to `nbest_entities`; I followed the notation from existing version of the code.